### PR TITLE
Update docking station article to list which devices support which USB-C features

### PR DESCRIPTION
**Pull request by nathandyer**
Adds a chart that makes it easier for customers to determine which USB-C features are supported by their system.

### CODE REVIEW

**Comment by nathandyer** [comment]
To see a better preview of the change:

Screenshot from 2020-09-18 18-44-21

File: _articles/use-docking-station.md [hunk 1]
```diff
@@ -16,11 +16,45 @@ section: accessories
...
+<td style="height: 23px;">Yes</td>
+</tr>
+<tr style="height: 23px;">
+<td style="height: 23px;">darp6</td>
+<td style="height: 23px;">Yes</td>
+<td style="height: 23px;">Yes</td>
+</tr>
+<tr style="height: 23px;">
+<td style="height: 23px;">darp5</td>
+<td style="height: 23px;">No</td>
```
Comment (leviport) [inline _articles/use-docking-station.md:48]: darp5 does support USB-C charging. I use a 60W USB-C charger on the regular with my darp5.

**Review by storyaddict — CHANGES_REQUESTED** [review]
Change Darp5 to USB-C charging "yes." Per Leviport's feedback.

**Review by ahoneybun — APPROVED** [review]
LGTM

**Pull request merged**